docs/k8s-helm.md

```diff
@@ -6,12 +6,14 @@ import TabItem from '@theme/TabItem';
 You can deploy Timeplus Enterprise on a Kubernetes cluster with [Helm](https://helm.sh/).
 
 ## Prerequisites
-* Ensure you have Helm 3.12 + installed in your environment. For details about how to install Helm, see the [Helm documentation](https://helm.sh/docs/intro/install/)
-* Ensure you have [Kubernetes](https://kubernetes.io/) 1.25 or higher installed in your environment. We tested our software and installation process on Amazon EKS, and self-hosted Kubernetes. Other Kubernetes distributions should work in the similar way.
-* Ensure you have allocated enough resources for the deployment. For a 3-nodes cluster deployment, by default each `timeplusd` requires 2 cores and 4GB memory. You'd better assign the node with at least 8 cores and 16GB memory.
-* Network access to Docker Hub.
+
+- Ensure you have Helm 3.12 + installed in your environment. For details about how to install Helm, see the [Helm documentation](https://helm.sh/docs/intro/install/)
```

Comment:
I use Zed to edit the docs repo and apply auto-format on save. So for example - will be changed to * for bullets and remove/add blank lines. Let's focus on the content review first. Before merging the PR, let me use Zed to save/format the file again to ensure the consistent markdown style.

Reply:
Can you share your formatter config? I can configure my format to follow the rules, otherwise we need to do this everytime

Reply:
You can check if Zed support https://editorconfig.org/ or not.

Reply:
https://zed.dev/docs/languages/markdown#format it only mentioned it will use Prettier to format the markdown. Since you will remove the markdown in other repo, this is the only copy. After we address other issues, I will format once and we don't need to reformat everything next time.

Reply:
Thanks. Then I'll leave it to you to do the final format before merging